appveyor.yml

```diff
@@ -37,3 +37,4 @@ environment:
     - ruby_version: 24-x64
     - ruby_version: 25-x64
     - ruby_version: 26-x64
+    - ruby_version: 27-x64
```

Comment:
Do we even run AppVeyor, or can we drop this config?

Reply:
No no for me we have to keep it. Why would you want to remove it? :)

Reply:
It doesn't seem to be configured https://ci.appveyor.com/project/rspec/rspec-rails, and we don't seem to run it (example passed and merged pull request https://github.com/rspec/rspec-rails/pull/2267, see View Details).

No objections on keeping it, just wondering why we need it since we've divorced `rspec-rails` from other RSpec projects, and even updating common Travis files mostly doesn't apply.

Reply:
Yes your are right. A little bit of context I think https://github.com/rspec/rspec-rails/pull/1218#issuecomment-63087627

Reply:
Thanks for the clarification! Let's keep it until someone volunteers to properly add build support.